front/app/components/UI/MentionsTextArea/index.tsx

```diff
@@ -186,50 +170,42 @@ class MentionsTextArea extends PureComponent<Props, State> {
     return style;
   };
 
-  mentionDisplayTransform = (_id, display) => {
+  const mentionDisplayTransform = (_id, display) => {
     return `@${display}`;
   };
 
-  handleOnChange = (event) => {
-    this.props?.onChange?.(event.target.value, this.props.locale);
+  const handleOnChange = (event) => {
+    onChange?.(event.target.value, locale);
   };
 
-  handleOnFocus = () => {
-    this.props?.onFocus?.();
+  const handleOnFocus = () => {
+    onFocus?.();
   };
 
-  handleOnBlur = () => {
-    this.props?.onBlur?.();
+  const handleOnBlur = () => {
+    onBlur?.();
   };
 
-  setRef = () => {
-    if (
-      this.textareaElement &&
-      this.textareaElement.current &&
-      this.props.getTextareaRef
-    ) {
-      this.props.getTextareaRef(
-        this.textareaElement.current as HTMLTextAreaElement
-      );
+  const setRef = () => {
```

Comment:
It seems to me that this is really "getting" the reference? Since it's not changing anything (unless I'm missing something)? :thinking: Do you think "getRef" would make more sense?

Reply:
Or does using getTextareaRef() with the textareaElement.current also _set_ the ref at the same time? I'm a bit confused here.

Reply:
I think the idea here is that we are getting the text area ref and setting it as the ref for the `MentionsInput` component so the naming makes sense in a way 🤔 Still, I renamed it to `setMentionsInputRef` to make it more clear, I hope it helps.